src/swagger-specs/query-service.yaml

```diff
@@ -776,6 +779,311 @@ paths:
+
+        - name: assetId
+          description: A query or schedule ID that you want the subscription information for.
+          required: true
+          in: path
+          type: string
+        - name: alertType
+          description: This property describes the state of query execution that triggers an alert. The response will only include alert subscription information for alerts of this type. The alert type is one of three values (`start`, `success`, `failure`).
+          required: true
+          in: path
+          type: string
+
+      responses:
+        200:
+          description: A successful response returns the user email IDs and notification channels for a particular query and alert type.
+          schema:
+            $ref: '#/definitions/subscribers_for_alert_by_assetId_and_alertType'
+        400:
+          description: "Bad Request. An invalid `alertType` was provided in the URL."
+          schema:
+            $ref: '#/definitions/get_alert_subscription_for_asset_id_and_alert_type_bad_request_error_message'
+        401:
+          description: "Unauthorized"
+        404:
+          description: "No alert was found for the given `assetId`."
+          schema:
+            $ref: '#/definitions/get_alert_subscription_for_asset_id_and_alert_type_not_found_error_message'
+        500:
+          description: "Internal Server Error."
+    patch:
+        operationId: "patchAlert"
+        tags:
+          - "Alert subscriptions"
+        summary: Enable or disable an alert
+        description: Update the status of an alert to either `enable` or `disable` by providing a query or schedule ID and an alert type in the request path.
+        consumes:
+          - application/json
+        produces:
+          - application/json
+        parameters:
+          - $ref: '#/parameters/authorization'
+          - $ref: '#/parameters/x-gw-ims-org-id'
+          - $ref: '#/parameters/x-api-key'
+          - $ref: '#/parameters/x-sandbox-name'
+          - $ref: '#/parameters/x-request-id'
+          - $ref: '#/parameters/User-Agent'
+
+          - name: assetId
+            description: A query ID or a schedule ID.
+            required: true
+            in: path
+            type: string
+
+          - name: alertType
+            description: This property determines the state of query execution that triggers a user to receive an alert. The alert type is one of three values (`start`, `success`, `failure`).
+            required: true
+            in: path
+            type: string
+
+          - name: operation
+            in: body
+            description: The requested operation.
+            required: true
+            schema:
+              type: object
+              properties:
+                op:
+                  type: string
+                  description: The operation to be performed. The only accepted value is 'replace'. It replaces the current alerts' status with the value provided in the request.
```

Comment:
`replace` should be backticked instead of single quoted.

Shouldn't it be "alert's" (possessive singular)?